roles/openshift_ca/tasks/main.yml

```diff
@@ -147,6 +147,10 @@
     with_items: "{{ g_master_legacy_ca_result.files | default([]) | lib_utils_oo_collect('path') }}"
     delegate_to: "{{ openshift_ca_host }}"
     run_once: true
+  - copy:
+      src: "{{ openshift_additional_ca }}"
+      dest: "{{ openshift_ca_clientconfig_tmpdir.stdout }}/"
+    when: openshift_additional_ca is defined
```

Comment:
Yeah. OK, are you finding that it worked otherwise or do we need to move to ca-bundle.crt as @pgier pointed out? I'm honestly not sure how that part works.